build-system/status-page/comment.js

```diff
@@ -23,7 +23,8 @@ const fetch = require('node-fetch');
 const {getChannels, steps} = require('./common');
 const {log} = require('../common/logging');
 
-const [number, body, user] = process.argv.slice(2);
+const [number, user] = process.argv.slice(2);
+const body = process.env.BODY;
```

Comment:
For consistency, would it make sense to use the same destructure notation as in `incident.js`?

```js
const {BODY: body} = process.env;
```

Reply:
I don't know, it's just one property. I think this looks cleaner than destructuring.

Reply:
I kinda like Raghu's version but don't feel strongly one way or the other :P

Reply:
I might have to avoid destructuring altogether because it gives a `string | undefined` type... hope this isn't too bad?
```
const body = process.env.BODY || '';
const after = process.env.COMMENT_AFTER || '';
const before = process.env.COMMENT_BEFORE || '';
```

Reply:
Hmm. Perhaps it's worth reconsidering https://github.com/ampproject/amphtml/pull/35145#discussion_r665686465 and adopting `minimist`? An additional advantage is that it allows you to specify the type of each argument.

```js
const minimist = require('minimist');
const argv = minimist(process.argv.slice(2), {string: ['body', 'after', 'before']});
const {body, after, before} = argv;
```

Reply:
Can you use minimist for `process.env` variables?

Reply:
Aha, I completely missed this distinction. I'm not aware of a way to use it without also switching the invocation to use `process.argv`, which appears to be the opposite of what this PR is trying to do. 😃 

Methinks it's okay to proceed with `process.env.FOO || ''` for now and adopt a different parsing solution later (if you feel it's worth it).

Reply:
Ok!